md/docs/derby-0.6/models/backends.md

```diff
@@ -1,38 +1,43 @@
+Racer stores are backed by ShareDB, which is used to persist data, perform queries, keep a journal of all operations, and pub/sub operations and changes to queries. Currently, ShareDB has [two pub/sub adapters](https://github.com/share/sharedb#database-adapters): one for in memory and one for Redis based pub/sub. ShareDB supports in memory or MongoDB storage. The database adapter [ShareDBMongo](https://github.com/share/sharedb-mongo)
+is backed by a real Mongo database and full query support. ShareDB is written with support for additional database adapters in mind.
 
 Getting started with a single-process server and MongoDB:
 
 ```js
-var derby = require('derby');
-var livedb = require('livedb');
-var liveDbMongo = require('livedb-mongo');
+var racer = require('racer');
+var ShareDbMongo = require('sharedb-mongo');
 
-var db = liveDbMongo('mongodb://localhost:27017/test');
-var backend = livedb.client(db);
-var store = derby.createStore({backend: backend});
+var db = new ShareDbMongo({mongo: 'mongodb://localhost:27017/test'});
+var backend = racer.createBackend({db: db});
+var model = backend.createModel();
 ```
 
-The above examples use the in-process driver by default. In a production environment, you'll want to scale accross multiple frontend servers and support updating data in other processes, such as migration scripts and additional services. For this, you should use the LiveDB Redis driver. LiveDB requires Redis 2.6 or newer, since it uses Lua scripting commands.
+The above examples use the in-process driver by default. In a production environment, you'll want to scale across multiple frontend servers and support updating data in other processes, such as migration scripts and additional services. For this, you should use the [ShareDB Redis pub/sub adapter](https://github.com/share/sharedb-redis-pubsub). ShareDB requires Redis 2.6 or newer, since it uses Lua scripting commands.
 
 ```js
-var derby = require('derby');
-var livedb = require('livedb');
-var liveDbMongo = require('livedb-mongo');
+var racer = require('racer');
 var redis = require('redis');
+var ShareDbMongo = require('sharedb-mongo');
+var RedisPubSub = require('sharedb-redis-pubsub');
+
 
-var db = liveDbMongo('mongodb://localhost:27017/test');
 // Two Redis clients are needed, since Redis requires separate clients for
 // pub/sub commands and other Redis commands
-var redisClient = redis.createClient();
-var redisObserver = redis.createClient();
-var driver = livedb.redisDriver(db, redisClient, redisObserver);
-var backend = livedb.client({snapshotDb: db, driver: driver});
-var store = derby.createStore({backend: backend});
+var pubsub = new RedisPubSub({
+  client: redis.createClient()
```

Comment:
in the simpler form, you don't need to instantiate redis yourself. you can just pass in the options and `sharedb-redis-pubsub` will do the intantiation